Error in GitHub Actions

```
Run npm test
  npm test
  shell: /usr/bin/bash -e {0}

> race-pace-calculator@1.0.0 test
> vitest run

failed to load config from /home/runner/work/race-pace-calculator/race-pace-calculator/vite.config.ts

⎯⎯⎯⎯⎯⎯⎯ Startup Error ⎯⎯⎯⎯⎯⎯⎯⎯
Error [ERR_REQUIRE_ESM]: require() of ES Module /home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vite/dist/node/index.js from /home/runner/work/race-pace-calculator/race-pace-calculator/vite.config.ts not supported.
Instead change the require of index.js in /home/runner/work/race-pace-calculator/race-pace-calculator/vite.config.ts to a dynamic import() which is available in all CommonJS modules.
    at _require.extensions.<computed> [as .js] (file:///home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vitest/node_modules/vite/dist/node/chunks/config.js:35931:9)
    at Object.<anonymous> (/home/runner/work/race-pace-calculator/race-pace-calculator/vite.config.ts:36:19)
    at _require.extensions.<computed> [as .js] (file:///home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vitest/node_modules/vite/dist/node/chunks/config.js:35930:44)
    at loadConfigFromBundledFile (file:///home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vitest/node_modules/vite/dist/node/chunks/config.js:35934:15)
    at async bundleAndLoadConfigFile (file:///home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vitest/node_modules/vite/dist/node/chunks/config.js:35808:17)
    at async loadConfigFromFile (file:///home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vitest/node_modules/vite/dist/node/chunks/config.js:35775:42)
    at async resolveConfig (file:///home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vitest/node_modules/vite/dist/node/chunks/config.js:35424:22)
    at async _createServer (file:///home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vitest/node_modules/vite/dist/node/chunks/config.js:25365:67)
    at async createViteServer (file:///home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vitest/dist/chunks/cli-api.BkDphVBG.js:6911:17)
    at async createVitest (file:///home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vitest/dist/chunks/cli-api.BkDphVBG.js:10202:17)
    at async prepareVitest (file:///home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vitest/dist/chunks/cli-api.BkDphVBG.js:10541:14)
    at async startVitest (file:///home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vitest/dist/chunks/cli-api.BkDphVBG.js:10488:14)
    at async start (file:///home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vitest/dist/chunks/cac.Cb-PYCCB.js:1412:15)
    at async CAC.run (file:///home/runner/work/race-pace-calculator/race-pace-calculator/node_modules/vitest/dist/chunks/cac.Cb-PYCCB.js:1391:2) {
  code: 'ERR_REQUIRE_ESM'
}

Error: Process completed with exit code 1.
```

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -5,6 +5,9 @@ on:
     branches: [main]
   workflow_dispatch:
 
+env:
+  FORCE_JAVASCRIPT_ACTIONS_TO_NODE24: true
+
 permissions:
   contents: read
   pages: write
@@ -19,7 +22,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: "18"
+          node-version: "22"
           cache: "npm"
 
       - name: Install dependencies

